guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -193,7 +193,9 @@ To move on from submitting bugs to helping resolve existing issues or contributi
 
 If you're a member of an organization that has codespaces enabled, you can fork Rails into that organization and use codespaces on GitHub. The Codespace will be initialized with all required dependencies and allows you to run all tests.
 
-If you're not a member of an organization that has codespaces enabled you can use the [VS Code remote containers plugin](https://code.visualstudio.com/docs/remote/containers-tutorial). The plugin will read the `.devcontainer` configuration in the repository and build the Docker container locally.
+#### Using VS Code remote containers
+
+If you have [Visual Studio Code](https://code.visualstudio.com) and [Docker](https://www.docker.com) installed you can use the [VS Code remote containers plugin](https://code.visualstudio.com/docs/remote/containers-tutorial). The plugin will read the [`.devcontainer`](../../.devcontainer) configuration in the repository and build the Docker container locally.
```

Comment:
To the best of my knowledge, links relative to the repo's directory structure won't be resolved when rendering at https://guides.rubyonrails.org/.

```suggestion
If you have [Visual Studio Code](https://code.visualstudio.com) and [Docker](https://www.docker.com) installed, you can use the [VS Code remote containers plugin](https://code.visualstudio.com/docs/remote/containers-tutorial). The plugin will read the [`.devcontainer`](https://github.com/rails/rails/tree/main/.devcontainer) configuration in the repository and build the Docker container locally.
```

Reply:
you are right, I hadn't seen it.